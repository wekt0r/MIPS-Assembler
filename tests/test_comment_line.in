add $s0, $s1, $s2 # here is comment after code so it should be in the output
#this one should not be in output
addi $t0, $s3, -12

addi $t0, $s3, -12


