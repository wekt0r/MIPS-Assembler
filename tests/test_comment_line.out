.text
00000000	02328020	add	$s0,$s1,$s2 # here is comment after code so it should be in the output
00000004	2268FFF4	addi	$t0,$s3,-12
00000008	2268FFF4	addi	$t0,$s3,-12
